Task 30 is completed

ViktoriaPa opened the pull request.

ViktoriaPa committed:
Task 30 is completed

shukal94 commented on src/test/java/seleniumWebDriver/AllLocators.java:
```diff
@@ -0,0 +1,92 @@
+
+import java.time.Duration;
+
+public class AllLocators {
```
this class contains duplicating steps and validations for login
let's create class Locators which will contain By constants, it could be useful for LoginTest

shukal94 commented on src/test/java/seleniumWebDriver/AllLocators.java:
```diff
@@ -0,0 +1,92 @@
+    void setup () {
+        driver = new ChromeDriver();
+        driver.get("https://mail.yandex.com");
+        WebElement logInButton = driver.findElement(By.xpath("//a[@href='https://passport.yandex.com/auth?retpath=https%3A%2F%2Fmail.yandex.com']"));
```
is there some more meaningful locator?

shukal94 reviewed (changes requested):
1) assertion messages are mandatory
2) remove all the unused imports
3) test names should reflect what do they testing, like `testXpathSelectorRecognizedCorrect`

ViktoriaPa committed:
made minor changes
1) added assertion messages
2) renamed tests
3) added LocatorsConstants and MainConstants classes

ViktoriaPa committed:
AllLocators class was deleted
